src/components/Button.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+
+export interface ButtonProps {
+  callback?: (event: React.MouseEvent) => void;
+  content?: string | JSX.Element;
+  className?: string;
+  disabled?: boolean;
+  type?: string;
+  href?: string;
+}
+
+export default class Button extends React.Component<ButtonProps, {}> {
+
+  constructor(props: ButtonProps) {
+    super(props);
+  }
+
+  render(): JSX.Element {
+   let content = this.props.content || "Submit";
+   let className = `btn ${this.props.className || "btn-default"}`;
+   let element = this.props.callback ?
+    this.renderButton(content, className) :
+    this.renderLink(content, className);
+
+   return element;
+ }
+
+ renderButton(content: string | JSX.Element, className: string): JSX.Element {
+   return (
+     <button
+      type={this.props.type || "submit"}
+      className={className}
+      disabled={this.props.disabled || null}
+      onClick={this.props.callback}
+     >
+      {content}
+     </button>
+   );
+ }
+
+ renderLink(content: string | JSX.Element, className: string): JSX.Element {
+   return (
+     <a
```

Comment:
Why and when would you want an anchor to be styled like a button? It's weird to have something like `<Button>` render an anchor.